src/libopenrave/kinbodygeometry.cpp

```diff
@@ -1031,6 +1053,22 @@ void KinBody::GeometryInfo::DeserializeJSON(const rapidjson::Value &value, const
                 _meshcollision.Clear();
             }
         }
+        if (value.HasMember("negativeCropContainerMargins")) {
+            orjson::LoadJsonValueByKey(value, "negativeCropContainerMargins", vGeomDataTemp);
+            vGeomDataTemp *= fUnitScale;
+            if (vGeomDataTemp != _vNegativeCropContainerMargins) {
+                _vNegativeCropContainerMargins = vGeomDataTemp;
+                _meshcollision.Clear();
```

Comment:
You do not need to clear mesh here since this parameter does not affect mesh generation.

Reply:
ok, they are removed.